packages/code/src/types.ts

```diff
@@ -47,5 +49,44 @@ export interface SyntaxProps extends React.HTMLAttributes<HTMLElement> {
   /**
    * An array of lines to highlight. The array can only contain numbers corresponding to the line numbers to highlight, and / or tuples representing a range (e.g. `[6, 10]`);
    */
-  highlightLines?: Array<number | [number, number]>;
+  highlightLines?: LineHighlightingDefinition;
+}
+
+export interface CodeProps extends Omit<SyntaxProps, 'onCopy'> {
```

Comment:
If we're extending SyntaxProps, why redefine overlapping props? We can probably remove explicit definitions for highlightLines and showLineNumbers.

Also, is there a reason that we're omitting onCopy from Syntax? We're not defining it more explicitly below.

Reply:
This is all pulled from the previous implementation of the component IIRC. You're probably right about removing those two definitions.

If I'm remembering right, `onCopy` being omitted and redefined is due to SyntaxProps including a definition for the native version of this handler. Once again, not something I added or changed aside from moving this code.

Reply:
Done with that first bit!

Reply:
Makes sense, it's easier to detect these things now that they're colocated :)

Reply:
> Another really small nit but while you're updating the website based on these changes, you can set showLineNumbers in the CodeDocs page

Done